bin/generate_config.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+import argparse
```

Comment:
Could you add a docstring explaining in 1-2 sentences what these config filese are and maybe link to a corresponding section on the 10x website?